### ISSUE TITLE
base64_cipher.py does not yet support Unicode

### ISSUE THREAD

**Issue by fbigun** [TheAlgorithms/Python#1027]
https://github.com/TheAlgorithms/Python/blob/b79a197e8c05251cf9443cbc5c15bb66ae23f3d8/ciphers/base64_cipher.py#L1

```
root@localhost$ python3
Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> s="woshi谁"
>>> s
'woshi谁'
>>> encodeBase64(s)
'd29zaPUBA==='
```

**Comment by poyea** [comment]
I'm guessing these functions do not work for Chinese characters.

**Comment by cclauss** [comment]
@fbigun Can you please edit the title of this issue to __base64_cipher.py does not yet support Unicode__ so that we have a feature request?  Python 2 support is no longer required.

**Comment by fbigun** [comment]
@cclauss 
The title has been changed

**Comment by laurent-pck** [comment]
I made the PR #1316 for this. I hope that using bytes() and str() is ok for you.

**Comment by cclauss** [comment]
Closed in #1316
